modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/rule-builder/FormsRuleList.es.js

```diff
@@ -304,45 +306,45 @@ const ListItem = ({dataProvider, fields, onDelete, onEdit, pages, rule}) => {
 						{Liferay.Language.get('if')}
 					</b>
 					{conditions.map((condition, index) => (
-						<>
+						<React.Fragment key={`fra${index}`}>
 							<Condition key={index} {...condition} />
 							{conditions.length - 1 > index && (
 								<LogicalOperator
-									key={'lo' + index}
+									key={`lo${index}`}
 									logicalOperator={
 										LOGICAL_OPERATOR[
 											rule['logical-operator']
 										]
 									}
 								/>
 							)}
-						</>
+						</React.Fragment>
 					))}
 					<br />
 					{actions.map(({action, ...otherProps}, index) => {
 						const Action = ACTIONS[action];
 
 						return (
-							<>
+							<React.Fragment key={`fra${index}`}>
```

Comment:
Why are we placing `key` here?

Reply:
Hi @diegonvs, I'm getting warning on test console about key attr. Do you know other option to prevent this warnings ?